.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@17media/17media",
+  "extends": "@17media/eslint-config-17media",
```

Comment:
Why change this? It does't work?

Reply:
https://github.com/17media/style-guide#installation--eslint-setup
No, my linter didn't run correctly with `@17media/17media`